### PR TITLE
Replace player numbers with tree species and use player board images as background.

### PR DESCRIPTION
**Pull request by fonglh**
See https://boardgamegeek.com/thread/1840832/okay-so-what-are-trees-represented

### CODE REVIEW

**Comment by sohymg** [comment]
i am not sure this is a good idea. makes it harder to associate the colour with the player?

**Comment by fonglh** [comment]
I think the primary way people associate players in this game is by colour anyway. Using actual names gives a bit more visual differentiation in the modal vs `Player x`.

image

**Comment by sohymg** [comment]
i mean in the sense when programming, perhaps add player constants like `const pine_player = 1`

**Comment by fonglh** [comment]
screen shot 2018-07-30 at 1 37 32 pm

Backgrounds for modal selection, light point table cells and board cells.

File: index.html [hunk 1]
```diff
@@ -132,13 +139,14 @@
       }
 
       .select-player-modal .select-player-btn {
-        width: 150px;
-        height: 100px;
+        width: 146px;
```
Comment (fonglh) [inline index.html:142]: Images are 1460px by 1050px.

**Pull request merged**